.tekton/tasks/echo-env.yaml

```diff
@@ -16,11 +22,18 @@ spec:
       image: node:$(params.node-version)
       script: |
         #!/bin/bash
+        ARTIFACTS_PATH="$(workspaces.output.path)"
+        cd $ARTIFACTS_PATH
 
-        echo "Node: $(node --version)"
+        if [ "$(params.prerelease)" == "true" ]; then
+          bin/install-prerelease.sh "$(params.prerelease-node-version)"
+          source "$HOME/.nvm/nvm.sh"
+          nvm use "$(params.prerelease-node-version)" > /dev/null 
+        fi
 
         if [ -n "$(params.npm-version)" ]; then
           npm install npm@$(params.npm-version) -g
         fi
 
-        echo "NPM: $(npm --version)"
+        echo "Using node: $(node --version)"
+        echo "Using NPM: $(npm --version)"
```

Comment:
issue: https://cloud.ibm.com/devops/pipelines/tekton/c2cd6a8d-ea5a-47b0-913e-cd172d63833f/runs/866cccf8-eddb-4bb6-b950-3a76a0a7f0f7?env_id=ibm:yp:eu-de

echo env task shows Node v20 $(node --version)

Reply:
This is the listener pipeline you're checking. prerelease attribute is false in that pipeline as of now. Could you please check the prerelease pipeline?

Reply:
I went via the last commit on this PR? Why is that different?

It works when I go to the listener 👍

Reply:
Ah because the PR listener does not set prerelease true.

Reply:
>  went via the last commit on this PR? Why is that different?

> 

We get the PR listener link from here, but the branch is configured in the prerelease pipeline. Currently, we don't want to run this in PR listener because the test suits